.changes/unreleased/🐛 Bug Fix-20221026-222227.yaml

```
@@ -0,0 +1,9 @@
+---
+kind: 🐛 Bug Fix
+body: use correct log prefixes which improves integration with GitHub Actions platform
```

@andrii-zakurenyi cc @mariiatuzovska  the comments here are specifically meant to communicate to the public, not developers. So it's a small difference, but think of this as a full description you'd send in an email or release notification to the wider audience. Adding suggested tweak for example.

We'll batch those up into a changelog when we are ready to do another release.

```suggestion
body: Adjust log prefix used in the GitHub action to improve the integration with actions and error identification.
```